services/management/ia/alerts_service.go

```diff
@@ -116,12 +116,19 @@ func (s *AlertsService) ListAlerts(ctx context.Context, req *iav1beta1.ListAlert
 				return err
 			})
 			if e != nil {
+				// Only FindRuleByID from transaction above can return codes.NotFound
```

Comment:
This comment should be rephrased. What did you wanted to say here?

Reply:
The transaction above has two operations `models.FindRuleByID` and `models.FindChannelsByIDs`. Only first one can return NotFound error, so we can log message about a `rule`, not about `rule or channel`

Reply:
What about this? The codes.NotFound code can be returned just only by the FindRulesByID method/function (used above).

Reply:
To be honest, I don't see much difference, but ok. Done.